sdk/lib/atl/atlconv.h

```diff
@@ -0,0 +1,312 @@
+/*
+ * PROJECT:     ReactOS ATL
+ * LICENSE:     LGPL-2.0-or-later (https://spdx.org/licenses/LGPL-2.0-or-later)
+ * PURPOSE:     String conversion
+ * COPYRIGHT:   Copyright 2023 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+
+#ifndef __ATLCONV_H__
+#define __ATLCONV_H__
+
+#pragma once
+
+#include "atlbase.h"
+
+namespace ATL
+{
+
+template <int t_nBufferLength = 128>
+class CA2CAEX
+{
+public:
+    LPCSTR m_psz;
+
+    CA2CAEX(_In_z_ LPCSTR psz) : m_psz(psz) { }
+
+    CA2CAEX(_In_z_ LPCSTR psz, _In_ UINT nCodePage) : m_psz(psz)
+    {
+        UNREFERENCED_PARAMETER(nCodePage);
+    }
+
+    ~CA2CAEX() throw() { }
```

Comment:
I'd add a comment both here and in the ctor that this class does not _own_ the string, so there is nothing to free here.

Reply:
Added comments.